File: LLM-Translations/GPT-4/codenet/C++/Go/s754383079.go

package main

import (
	"fmt"
	"strings"
)

const M = 1010

func main() {
	var a, b string
	var n, m, ans int
	ans = M + 10

	fmt.Scan(&a)
	n = len(a)
	a = " " + a
	fmt.Scan(&b)
	m = len(b)
	b = " " + b

	for i := 1; i+m-1 <= n; i++ {
		sum := 0
		for j := 1; j <= m; j++ {
			if a[i+j-1] != b[j] {
				sum++
			}
		}
		if sum < ans {
			ans = sum
		}
	}
	fmt.Printf("%d\n", ans)

}
// 

File: LLM-Translations/GPT-4/codenet/C++/Python/s754383079.py

def main():
    a = input().strip()
    b = input().strip()
    n, m = len(a), len(b)
    ans = float('inf')
    
    for i in range(n - m + 1):
        sum_ = sum(ai != bj for ai, bj in zip(a[i:], b))
        ans = min(ans, sum_)
    
    print(ans)

if __name__ == '__main__':
    main()


